dynamic/info.go

```diff
@@ -43,6 +45,19 @@ func providerInfo(ctx context.Context, p run.Provider, value parameterize.Value)
 			Path: "", Data: tfbridge.ProviderMetadata(nil),
 		},
 
+		Python: &tfbridge.PythonInfo{
+			PyProject:            struct{ Enabled bool }{true},
+			RespectSchemaVersion: true,
```

Comment:
You don't actually need to set RespectSchemaVersion. Having something in Parameterization tells codegen to require a version and embed it anyway.